taipy/gui/builder/_api_generator.py

```diff
@@ -46,17 +47,18 @@ def add_default(self):
         module_name = current_frame.f_back.f_globals["__name__"]
         self.__module = module = sys.modules[module_name]
         with open(os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "viselements.json"))) as viselements:
-            data = json.load(viselements)
-            if "blocks" not in data or "controls" not in data:
+            viselements = json.load(viselements)
```

Comment:
Is it not a bit confusing to re-use viselements?

Reply:
will do! tx Fred